summary.php

```diff
@@ -188,14 +188,14 @@
 	<title>Resources Module - <?php echo $pageTitle; ?></title>
 	<link rel="stylesheet" href="css/style.css" type="text/css" media="print" />
 	<link rel="stylesheet" href="css/style.css" type="text/css" media="screen" />
-	<link rel="SHORTCUT ICON" href="images/butterflyfishfavicon.ico" />
+	<link rel="SHORTCUT ICON" href="images/favicon.ico" />
 
 	</head>
 	<body>
 
 
 	<div class='printContent'>
-	<table class='linedFormTable'>
+	<table class='linedFormTable' style='width:510px;'>
```

Comment:
I checked in with our CSS designer, and he said that since he only wanted to quickly update this particular table, he simply followed the same convention of inline style that was already in use across the code.